lib/cog/commands/alias/create.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Commands.Alias.Create do
     -h, --help  Display this usage info
 
   EXAMPLE
-    alias create my-awesome-alias "echo \"My awesome alias\""
+    alias create my-awesome-alias \"echo \"My awesome alias\"\"
```

Comment:
I don't think escaping those quotes are necessary here, although I think Elixir renders them just the same inside a triple quote block.